### PR TITLE
Update -f info to display how to run multiple rule files

### PR DESCRIPTION
**Pull request by adamkvitek**
Linear: https://linear.app/r2c/issue/MKT-316/update-semgrep-help-and-cli-docs-with-f-option

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/cli-usage.md [hunk 1]
```diff
@@ -79,6 +79,11 @@ Options:
                                   rules tailored to this project; your project
                                   URL will be used to log in to the Semgrep
                                   registry.
+
+                                  To run multiple rule files simultaneously, use -f
+                                  before every YAML, URL, or Semgrep registry
+                                  entry name. For example:
+                                  `semgrep -f your/java.net.url. -f path/secrets`.
```
Comment (s-santillan) [inline docs/cli-usage.md:86]:
```suggestion
                                  `semgrep -f your/java.net.url. -f config/rule.yml
```

**Review by s-santillan — APPROVED** [review]
LGTM, minor nit, non-blocking, feel free to ignore my suggestion:

For the examples, I removed thke extra (.) on the URL address AND used a more common path to rules that previous users have used ([folder structure here](https://github.com/returntocorp/semgrep/issues/4574))

File: docs/cli-usage.md (hunk 1 above)
Comment (ievans) [inline docs/cli-usage.md:86]:
```suggestion
                                  `semgrep --config p/python --config myrules/myrule.yaml`.
```
Prefer to use long flags in docs, IMO. also I think it's useful to provide a registry example

Comment (adamkvitek) [reply docs/cli-usage.md]: Thanks! Good point.

Comment (adamkvitek) [reply docs/cli-usage.md]: Ok, actually, now I am confused - so we can run more rules by using any command again? @ievans

Comment (minusworld) [reply docs/cli-usage.md]: @ievans Updated in CLI too: https://github.com/returntocorp/semgrep/pull/4751

Comment (adamkvitek) [reply docs/cli-usage.md]: @ievans @minusworld updated here in the docs as well.

**Pull request merged**